.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -621,7 +626,27 @@ def write(x=''):
 
 ##############################################################################
 # Write submitty json
+user_id_requirements = {
+    "any_user_id": True,
+    "require_name": False,
+    "min_length": 6,
+    "max_length": 25,
+    "name_requirements": {
+        "given_first": False,
+        "given_name": 2,
+        "family_name": 4
```

Comment:
can we add a comment explaining what this means?  THis seems to suggest that every users family/last name must be at least 4 characters.  (And there are common family names with just 2 or 3 characters.). But from skimming this code, this isn't a 'requirement' its how the user id is made.